### DISCUSSION TITLE
Issue with mysql transaction lock

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#2904] [Help with using Postal]
Hello everyone

i am using non docker postal. but having issue with lock transaction

i have already defined settings in mariadb
--------------------------------
[mariadb]
max_connections = 1000
innodb_buffer_pool_size=100M
max_allowed_packet=1024M
net_read_timeout=3600
net_write_timeout=3600
innodb_flush_method=littlesync
innodb_use_native_aio=OFF
log_bin=ON
--------------------------------
<img width="545" alt="image" src="https://github.com/postalserver/postal/assets/32220165/bce6bbb9-599e-40b3-b042-a2ff5c7201d5">

i have **10000** emails in queue_messages table, 

i am trying to run 
**update queued_messages set locked_by = NULL, locked_at = NULL, retry_after = NULL, ip_address_id = NULL;**
but it got transaction lock

FYI: i have this configuration of the server
--------------------------------
Product/Service: High End Server - Octa Core E5 2667 v4 (16vCPU) 
Operating System: Ubuntu 20.04 -64 Bit
Memory: 32 GB ECC
Primary HDD: 960 GB SSD
----------------------------------

**Comment by willpower232** [comment]
What error message are you seeing? Can you share a screenshot?

**Comment by rccess** [comment]
<img width="529" alt="image" src="https://github.com/postalserver/postal/assets/32220165/6b0a9fd4-2c81-47e6-871c-78adc7e85eea">
[worker.2:3110] [2024-04-02T06:22:21.455] INFO -- : [SCO5JN8I] IOError: closed session
(Net::OpenTimeout: execution expired)

**Reply by willpower232** [reply]
These are network errors, not database errors.

> IOError: closed session

there was some kind of network interruption with whoever was supposed to receive the message udring some amount of transfer

> Net::OpenTimeout: execution expired

there was some kind of error making the connection, potentially missing, invalid, or unreachable DNS, or perhaps you are blocked

Do the messages eventually get through?
